### PR TITLE
Added Docs for List

### PR DESCRIPTION
**Pull request by rohanthacker**
Brief Documentation for the List Component

### CODE REVIEW

File: src/list/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Lists present multiple line items vertically as a single continuous element.
+
+
+### To include the MDL **list** component:
+&nbsp;1. Code a `<ul>` element with the class `mdl-list`; this is the "outer" container, intended to hold all of the list's content.
+```html
+<ul class='mdl-list'>
+</ul>
+```
+&nbsp;2. Code as many `<li>` elements as required with the class `mdl-list__item`; this is the individual item that a list is comprises off.
```
Comment (surma) [inline src/list/README.md]: *comprised*

File: src/list/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+```html
+<ul class='mdl-list'>
+<li class="mdl-list__item"></li>
+<li class="mdl-list__item"></li>
+<li class="mdl-list__item"></li>
+</ul>
+NOTE : Stop here if you a simple single line list
```
Comment (surma) [inline src/list/README.md]: if you *have* a

File: src/list/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+&nbsp;3. Append the class `mdl-list__item-two-line` or`mdl-list__item-three-line`, if you require a Two Line List or Three Line List.
+```html
+<ul class='mdl-list'>
+<li class="mdl-list__item mdl-list__item-two-line"></li>
+<li class="mdl-list__item mdl-list__item-two-line"></li>
+<li class="mdl-list__item mdl-list__item-two-line"></li>
+</ul>
+```
+
+&nbsp;4. Add the appropriate classes to style your content, ( Avatar, Icon, Primary and secondary content)
```
Comment (surma) [inline src/list/README.md]: s/,/./

File: src/list/README.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+## Configuration options
+
+The MDL CSS classes apply various predefined visual enhancements to the list. The table below lists the available classes and their effects.
+
+MDL class
+
+| MDL Class        | Effect           | Remark  |
+| ------------- |:-------------:| -----:|
+| .mdl-list      | Defines list as an MDL component| - |
+| .mdl-list__item      | Defines a the single list item  | - |
```
Comment (surma) [inline src/list/README.md]: ~~the~~

File: src/list/README.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+| .mdl-list__item-primary-content | Defines the primary content area      |    $1 |
+| .mdl-list__item-avatar | Defines the avatar in a List Item      |    $1 |
+| .mdl-list__item-two-line | Defines the List Item as Two Line      |    $1 |
+| .mdl-list__item-three-line | Defines the List Item  as a Three Line      |    $1 |
+| .mdl-list__item-secondary-content | Defines the secondary content area      | Only on Three and Two Line |
+| .mdl-list__item-secondary-info | Defines the Info area of secondary content      | Only on Three and Two Line |
+| .mdl-list__item-secondary-action | Defines the Action area  of secondary content      | Only on Three and Two Line |
```
Comment (surma) [inline src/list/README.md]: ~~[Double Space]~~

**Comment by surma** [comment]
Awesome work, thanks a lot @rohanthacker! Thanks for getting back to us on this.

A few typos, after that, this LGTM :)

**Comment by rohanthacker** [comment]
@surma :+1: Thank you too for the work you and rest of the team put in to product such a high quality product.

**Comment by surma** [comment]
cc @Garbee @addyosmani @sgomes re your question.

I’d at least make it “content *type* modification class”. Otherwise I am okay with the terminology.

**Comment by rohanthacker** [comment]
Will open a new PR in place of this. Made an error.

**Pull request closed without merging**